### DISCUSSION TITLE
brew uses --installed python@3.8 reports something wrong

### DISCUSSION THREAD

**Discussion by josser** [Homebrew/discussions#106] [Everyday usage] [answered]
Hi! 
I'm trying to avoid having too much packages installed, and today I've found that I have both python 3.8 and 3.9 installed.
So, I've decided to understand which packages require python@3.8 and found this command in docs: 

```brew uses --installed python@3.8 ```
It reports this list:

cairo - https://github.com/Homebrew/homebrew-core/blob/master/Formula/cairo.rb
harfbuzz - https://github.com/Homebrew/homebrew-core/blob/master/Formula/harfbuzz.rb
libass - https://github.com/Homebrew/homebrew-core/blob/master/Formula/libass.rb
midnight-commander - https://github.com/Homebrew/homebrew-core/blob/master/Formula/midnight-commander.rb

but neither of them have 'python' in 'depends_on' section. 
So, I guess 'brew uses' resolve python@3.8 in sub-dependencies. 
If this is correct guess, is there any way to found this deps? 
Thank you!

**Comment by SMillerDev** [comment]
You can run it with `--recursive` https://docs.brew.sh/Manpage#uses-options-formula

**Reply by josser** [reply]
@SMillerDev oh, I forget to tell. I've tried adding --recursive but it doesn't change results

**Comment by josser** [comment, marked as answer]
Ok, this is not the answer to original question, but I have found the way to get rid of python@3.8 
I've uninstalled midnight-commander, ffmpeg and all of their leaves (python@3.8 and python@3.9 was there) 
Than I've installed them back and no python@3.8 was in deps.
